vdom/src/com/vdom/core/QuickPlayPlayer.java

```diff
@@ -1083,4 +1083,9 @@ public boolean shouldAutoPlay_temple_cardsToTrash(MoveContext context) {
     	return false;
     }
 
+    public boolean shouldAutoPlay_smallCastle__shouldTrashSmallCastlePlayed(MoveContext context) { return false; }
+    public boolean shouldAutoPlay_smallCastle_castleToTrash(MoveContext context) { return false; }
+
+    public boolean shouldAutoPlay_sprawlingCastle_chooseOption(MoveContext context) { return true; }
```

Comment:
This should not be set to quick play - it depends on at the very least whether you have the Opulent Castle and whether you think you'll draw it together with the Estates.